lights.go

```diff
@@ -192,7 +184,7 @@ func (l *Light) Set(s *State) error {
 // State holds a structure that is used to update a light's state.
 type State struct {
 	// On, when true, will turn a light on.
-	On bool `json:"on,omitempty"`
```

Comment:
The problem here is that if you simply want to change the brightness of a light by doing something like:
```go
l.Set(&State{Brightness: 200})
```
The light will be switched off, because `On` defaults to `false`, and if `omitempty` isn't there, it will be sent as such.

I've chosen this way because I've figured that it would make more sense as an API. This is mentioned in the documentation of the [Set](https://godoc.org/github.com/gbbr/hue#Light.Set) method and is intentional.

I think that removing `omitempty` here is also a solution, which we would have to mention in the [documentation](https://godoc.org/github.com/gbbr/hue#State) of the `State` structure, but I think the more convenient method is the original one. 

What do you think?